core/benchmarks/CoalescedRDDBenchmark-jdk21-results.txt

```diff
@@ -2,39 +2,39 @@
-Coalesce Num Partitions: 1000 Num Hosts: 5              425            428           4          0.2        4251.4       0.6X
-Coalesce Num Partitions: 1000 Num Hosts: 10             263            267           3          0.4        2634.4       1.0X
-Coalesce Num Partitions: 1000 Num Hosts: 20             182            188           9          0.5        1822.5       1.5X
-Coalesce Num Partitions: 1000 Num Hosts: 40             142            143           1          0.7        1424.5       1.9X
-Coalesce Num Partitions: 1000 Num Hosts: 80             123            131           8          0.8        1226.2       2.2X
-Coalesce Num Partitions: 5000 Num Hosts: 1             7484           7491          10          0.0       74836.3       0.0X
-Coalesce Num Partitions: 5000 Num Hosts: 5             1873           1880          11          0.1       18725.7       0.1X
-Coalesce Num Partitions: 5000 Num Hosts: 10             995           1005          11          0.1        9950.1       0.3X
-Coalesce Num Partitions: 5000 Num Hosts: 20             557            561           6          0.2        5570.9       0.5X
-Coalesce Num Partitions: 5000 Num Hosts: 40             337            341           4          0.3        3369.7       0.8X
-Coalesce Num Partitions: 5000 Num Hosts: 80             222            223           1          0.5        2222.0       1.2X
-Coalesce Num Partitions: 10000 Num Hosts: 1           14102          14133          45          0.0      141020.7       0.0X
-Coalesce Num Partitions: 10000 Num Hosts: 5            3681           3702          31          0.0       36811.9       0.1X
-Coalesce Num Partitions: 10000 Num Hosts: 10           1871           1884          22          0.1       18706.8       0.1X
-Coalesce Num Partitions: 10000 Num Hosts: 20            998           1004           5          0.1        9980.9       0.3X
-Coalesce Num Partitions: 10000 Num Hosts: 40            570            575           5          0.2        5696.7       0.5X
-Coalesce Num Partitions: 10000 Num Hosts: 80            345            346           2          0.3        3447.7       0.8X
+Coalesce Num Partitions: 100 Num Hosts: 1               359            371          12          0.3        3586.9       1.0X
+Coalesce Num Partitions: 100 Num Hosts: 5               181            190          14          0.6        1812.1       2.0X
+Coalesce Num Partitions: 100 Num Hosts: 10              178            185           6          0.6        1779.9       2.0X
+Coalesce Num Partitions: 100 Num Hosts: 20              153            156           4          0.7        1531.2       2.3X
+Coalesce Num Partitions: 100 Num Hosts: 40              148            149           1          0.7        1479.1       2.4X
+Coalesce Num Partitions: 100 Num Hosts: 80              166            170           5          0.6        1657.8       2.2X
```

Comment:
The ratio slightly changed here in a marginal difference. Java 17 result is okay.